### PR TITLE
[WIP] MULTIARCH-941: Power Updates for OCP 4.8

### PR DESCRIPTION
**Pull request by ktania46**
OCP version: 4.8 and later

JIRA issue: https://issues.redhat.com/browse/MULTIARCH-941

Google review doc: 

- Restricted Access: https://docs.google.com/document/d/1iqvdZWSQlyJtv86ByROfCnYIXxCMv37MZuO-qYdScSc/edit?usp=sharing 

- Direct installation: https://docs.google.com/document/d/1Pt_WdN9z3TD6KoN_lDjduJYY9FALkX8PP-utuSSB97E/edit?usp=sharing

- Scale table: https://docs.google.com/document/d/1IZ4UI_4nXMNr85zPH5RmSB4BdkBVBfW1hF14W0fXsmI/edit?usp=sharing 

- Technology Preview Tracker:  https://docs.google.com/document/d/1vaGHKLpN56T2xeNLrGcciEbSOfN03optys32CXuK69I/edit?usp=sharing  

Reviewer: Manoj Kumar, Pravin D'Silva, Piyush Gupta

Preview pages: 

- Install : https://deploy-preview-31787--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power.html

- Restricted install :  https://deploy-preview-31787--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-restricted-networks-ibm-power.html

- Scale table : https://deploy-preview-31787--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/planning-your-environment-according-to-object-maximums.html#cluster-maximums-environment_object-limits

### CODE REVIEW

File: modules/openshift-cluster-maximums-environment.adoc [hunk 1]
```diff
@@ -56,3 +56,51 @@ AWS cloud platform:
...
 --
+
+IBM Power Systems platform:
+
+[options="header",cols="7*"]
+|===
+| Node |Flavor |vCPU |RAM(GiB) |Disk type|Disk size(GiB)/IOS |Count
+
+| Master/etcd ^[1]^
+| r5.4xlarge
```
Comment (manojnkumar) [inline modules/openshift-cluster-maximums-environment.adoc:67]: I do not know where these flavor names are coming from.  Can we drop this whole Flavor Column?

Comment (ktania46) [reply modules/openshift-cluster-maximums-environment.adoc]: @manojnkumar  I have replaced r5.4xlarge in flavor column with P9-e980 . That was mistake on my part. Hope this is fine, if you still want this column to be removed I can.

Comment (manojnkumar) [reply modules/openshift-cluster-maximums-environment.adoc]: > @manojnkumar I have replaced r5.4xlarge in flavor column with P9-e980 . That was mistake on my part. Hope this is fine, if you still want this column to be removed I can.

@ktania46 : Yes, I would prefer that we remove it.

**Comment by lsmcfadden** [comment]
- Scale table looks good - aligns with feedback.
- TP table contains some "?", suggest to make these "-" (if there is any mismatch between p and z (unsure if z answers are here) please bring that to my attention.
- For restricted install, I asked Pravin to confirm 3 node cluster (new text you added) matches our QE testing. Pls confirm with him.
- For direct install please address these comments (if not done already)
  - https://docs.google.com/document/d/1Pt_WdN9z3TD6KoN_lDjduJYY9FALkX8PP-utuSSB97E/edit?disco=AAAAMuGiVfw
  - https://docs.google.com/document/d/1Pt_WdN9z3TD6KoN_lDjduJYY9FALkX8PP-utuSSB97E/edit?disco=AAAAMuGiVgI
  - https://docs.google.com/document/d/1Pt_WdN9z3TD6KoN_lDjduJYY9FALkX8PP-utuSSB97E/edit?disco=AAAAMuGiVgo
  - https://docs.google.com/document/d/1Pt_WdN9z3TD6KoN_lDjduJYY9FALkX8PP-utuSSB97E/edit?disco=AAAAMuGiVgs
  - https://docs.google.com/document/d/1Pt_WdN9z3TD6KoN_lDjduJYY9FALkX8PP-utuSSB97E/edit?disco=AAAAMuGiVgw
  - https://docs.google.com/document/d/1Pt_WdN9z3TD6KoN_lDjduJYY9FALkX8PP-utuSSB97E/edit?disco=AAAAMuGiVhc
  - As above on 3 node cluster.

Once you confirm, I'll add a 'lgtm' on Wed.

**Comment by ktania46** [comment]
please refer https://github.com/openshift/openshift-docs/pull/33351

**Pull request closed without merging**